.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # default location of `.github/workflows`
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
+    target-branch: 'develop'
+    # assignees:
+    #   - assignee_one
+    # reviewers:
+    #   - reviewer_one
```

Comment:
If you decide that certain people on your team should take care of the PRs that Dependabot creates, use these two attributes to automatically set personnel respectively.

(also lines 22-25)